test/unit/api/api.spec.js

```diff
@@ -602,6 +602,45 @@ describe('endpoint unit test', () => {
             });
         });
       });
+
+      test('should retrieve stars list with credentials', async (done) => {
```

Comment:
This should be inside of ` describe('should test star api', () => {` otherwise report looks odd

```
 should test publish/unpublish api
        ✓ should publish a new package with no credentials (35ms)
        ✓ should retrieve stars list with credentials (53ms)
        ✓ should unpublish a new package with credentials (47ms)
        ✓ should fail due non-unpublish nobody can unpublish (22ms)
        ✓ should be able to publish/unpublish by only super_admin user (47ms)
        ✓ should be able to publish/unpublish by any user (55ms)
        should test star api
          ✓ should star a package (10ms)
          ✓ should unstar a package (10ms)
```